### ISSUE TITLE
Chaining with contents renders inproperly.

### ISSUE THREAD

**Issue by armoredpigeon** [JoeChapman/svg-builder#14]
When I chain two groups together, each containing other svg elements, instead of getting 
`<g id="1"><circle></g><g id="2"><rect></g>`
I get it all wrapped inside the second group, like this:
`<g id="2"><g id="1"><circle></g><circle></g>`

**Comment by JoeChapman** [comment]
You are correct, I have fixed this issue and released it in patch version 3.0.1
